cds_migrator_kit/rdm/migration/stats/event_generator.py

```diff
@@ -96,13 +118,17 @@ def process_download_event(entry, rec_context, logger):
         "recid": _record_version["new_recid"],
         "parent_recid": rec_context["parent_recid"],
         "referrer": None,
+        "is_robot": is_bot,
         "via_api": False,
-        "is_robot": entry.get("bot", False),
         "country": entry.get("country", ""),
         "visitor_id": entry.get("visitor_id", ""),
         "unique_session_id": entry["unique_session_id"],
         # Note: id_bibrec doesn't have the new format pids
         "unique_id": f"ui_{_record_version['new_recid']}",
+        # Mark the event as migrated
+        "is_lcds": True,
+        # Mark if event was COUNTER compliant
+        "before_COUNTER": before_COUNTER,
```

Comment:
did we say that we add it only when `before_COUNTER = true`?
New events will not have this field at all.

Reply:
Discussed IRL to keep it for differentiating between COUNTER compliant migrated events